README.md

```diff
@@ -13,34 +13,33 @@ Currently supports
  - Coub
  - Wistia
 
-# grunt
-
-```shell
-$ grunt
-$ grunt build
-$ grunt test
-$ grunt dist
-
-#build and test specific plugins
-$ grunt --youtube --vimeo
```

Comment:
Is it still possible to build the dist for specific providers? People asked for it before so I'd like to keep it if possible. I'm fine with moving all the test stuff to npm but the grunt building part could be used in parallel?

Reply:
It is if you are using webpack, browserify, or something similar (anything that supports require - very common today). You import `js-video-url-parser/lib/base` first and then each plugin such as `js-video-url-parser/lib/youtube`. You don't need to build the project itself anymore. You simply include the parts you want and only those plugins get included during your own projects build process.

You only need a build for bower, which needs a dist file on GitHub that people can include with a script tag. I use rollup to produce this build. Most people today are using the npm registry though, and they actually don't need the produced build.